skygo.py

```diff
@@ -213,17 +274,22 @@ def play(self, manifest_url, package_code, info_tag=None, apix_id=None):
                 li = xbmcgui.ListItem(path=manifest_url)
                 if info_tag:
                     li.setInfo('video', info_tag)
-                # Force smoothsteam addon
-                li.setProperty('inputstream.smoothstream.license_type', 'com.widevine.alpha')
+                # Inputstream settings
+                is_addon = getInputstreamAddon()
```

Comment:
Could be removed.
inputstream.smoothstream is actually obsolete and we could always use inputstream.adaptive